### ISSUE TITLE
Custom menu icon when importing a .json export

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#363]
```
When importing a JSON exported file from another site, the custom menu icon is 
broken.

It looks like we need to add bloginfo('url') when importing.

See attached image
```

Original issue reported on code.google.com by `kb6...@gmail.com` on 3 Apr 2012 at 10:59

Attachments:
- [Untitled.png](https://storage.googleapis.com/google-code-attachments/wordpress-custom-content-type-manager/issue-363/comment-0/Untitled.png)

**Comment by GoogleCodeExporter** [comment]
```
I can't remember if this is fixed on the dev version or not.  There's a lot of 
hoop jumping there because WP stores the full URL to the custom icon, but when 
you export your definition, you have to make the URL relative (e.g. 
http://oldsite.com/my_image/.jpg won't work -- it needs to be changed to 
http://newsite.com/my_image.jpg).  And a lot of that stuff relies on server 
functions, so I really need to know what server you're on.  Please copy full 
details from the bug report page inside any CCTM page in the manager.  To be 
thorough, you can try the dev version and see if this issue is corrected.
```

Original comment by `ever...@fireproofsocks.com` on 3 Apr 2012 at 3:15
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
This is fixed on dev branch, will go out in 0.9.12

Committed revision 536831.
```

Original comment by `ever...@fireproofsocks.com` on 26 Apr 2012 at 4:07
- Changed state: **Fixed**
- Added labels: **Milestone-Release0.9.6**
